types/office-js-preview/index.d.ts

```diff
@@ -11285,36 +11285,17 @@ declare namespace Office {
          */
         isAllDayEvent: boolean;
         /**
-         * Gets the Exchange Web Services item class of the selected item.
+         * Gets the Exchange Web Services item class of the selected appointment.
```

Comment:
@ndeleuze This change applies to the existing Office.AppointmentRead.itemClass property. I included this change to be consistent with the description of `getItemClassAsync` when it still applied to the Appointment Organizer surface. Now that it only applies to Message Compose, I'll move these changes to a separate PR to avoid confusion. Thanks for catching this!